cime_config/acme/machines/config_batch.xml

```diff
@@ -44,7 +44,7 @@
        <arg flag="--cwd" name="CASEROOT"/>
        <arg flag="-A" name="PROJECT"/>
        <arg flag="-t" name="JOB_WALLCLOCK_TIME"/>
-       <arg flag="-n" name="$TOTALPES/$PES_PER_NODE"/>
+       <arg flag="-n" name=" $TOTALPES/$PES_PER_NODE"/>
```

Comment:
Would you mind adding the comment from the cesm side about the space required at the beginning?